prow/config/jobs/istio.yaml

```diff
@@ -170,6 +170,11 @@ jobs:
       - name: TEST_SELECT
         value: "-multicluster"
 
+  - name: integ-helm-tests
+    type: presubmit
```

Comment:
We should add this as both presubmit and postsubmit to track flakes, etc like all of our other tests. and I recommend adding `modifiers: [hidden, optional]` for a while so we can ensure it stabilizes before we block every PR on it as we have not proven it is a stable test yet

Reply:
Makes sene, done

Reply:
I don't know how to add it for both pre & post submit. I deleted the type which based on README makes it run for both. What target should i use?